pyctuator/impl/aiohttp_pyctuator.py

```diff
@@ -0,0 +1,192 @@
+import dataclasses
+import json
+from collections import defaultdict
+from datetime import datetime
+from functools import partial
+from http import HTTPStatus
+from typing import Any, Callable, List, Mapping, Optional
+
+from aiohttp import web
+from multidict import CIMultiDict
+
+from pyctuator.impl.pyctuator_impl import PyctuatorImpl
+from pyctuator.impl.pyctuator_router import PyctuatorRouter
+from pyctuator.httptrace import TraceRecord, TraceRequest, TraceResponse
+from pyctuator.impl import SBA_V2_CONTENT_TYPE
+
+
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
```

Comment:
You can put the two "disables" in one line with commas between them.
Not critical to me